cpp/include/raft/linalg/distance_type.h

```diff
@@ -35,18 +35,18 @@ enum DistanceType : unsigned short {
   EucUnexpandedL2Sqrt = 5,
   /** basic inner product **/
   InnerProduct = 6,
-  /** ChebyChev (Linf) distance **/
-  ChebyChev = 7,
+  /** Chebyshev (Linf) distance **/
+  EucUnexpandedLinf = 7,
```

Comment:
Are there any existing callers using these old names?
I believe @lowener is about to switch over to them so tagging him here so he can see the new names.

Reply:
These names were put here explicitly for this PR so there shouldn't be any existing code using them.